cloud/aws/terraform/control/variables.tf

```diff
@@ -4,3 +4,7 @@ variable "region" {}
 variable "availability_zones" {}
 variable "ami" {}
 variable "key_name" {}
+
+variable "stack_name" {
+  default = "hashistack"
+}
```

Comment:
Do we need to define this default variable here considering the module default is the same from what I can see?

Reply:
Yeah it's the same, but If you don't provide a default here `terraform apply` will complain. 

And the reasoning for adding this variable here is to avoid conflicts when deploying the demo multiple times in the same AWS account. 

Maybe this should be [`random_pet`](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet)?